app/src/assets/app.extensions.json

```diff
@@ -401,7 +402,8 @@
         "title": "APP.ACTIONS.LEAVE",
         "icon": "exit_to_app",
         "actions": {
-          "click": "LEAVE_LIBRARY"
+          "click": "LEAVE_LIBRARY",
+          "focusedElementOnCloseSelector": "#app\\.toolbar\\.leaveLibrary"
```

Comment:
we are adding the notion of the application css and paths to the config file, not sure is a great idea

Reply:
done